pymc3/distributions/multivariate.py

```diff
@@ -1017,6 +1017,31 @@ def __init__(self, eta=None, n=None, p=None, transform='interval', *args, **kwar
         self.tri_index[np.triu_indices(n, k=1)] = np.arange(n_elem)
         self.tri_index[np.triu_indices(n, k=1)[::-1]] = np.arange(n_elem)
 
+    def _random(self, n, eta, size=None):
+        beta = eta + (n-1)/2
+        P = np.zeros((n, n)) # partial correlations
+        r_triu = []
+        
+        for k in range(n-1):
```

Comment:
I can not see an easy way to do it, the step below converting the partial correlation to raw correlation is especially tricky.